### PR TITLE
IPA + Licensing + Policy + Permission + Tenderly

### PR DESCRIPTION
**Pull request by edisonz0718**
Change:
1. Add integration tests for IPA, Licensing, Policy and Permission
2. Fix issues on these functions
3. Tenderly integration. Now use Tenderly for testing.
4. Remove .env variable for smart contracts

Policy integration test is working but is skipped due to Tenderly rewind issue. Need to deploy a fresh protocol to make it work.

Next:
1. Unit tests and more integration tests to cover corner cases
2. Comments
3. Error handling, More DevEx improvement
4. Convenient functions for creating social remix and commercial use policy.

### CODE REVIEW

File: packages/core-sdk/test/integration/license.test.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+          txOptions: {
+            waitForTransaction: waitForTransaction,
+          },
+        }),
+      ).to.not.be.rejected;
+
+      expect(response.txHash).to.be.a("string");
+      expect(response.txHash).not.empty;
+
+      if (waitForTransaction) {
```
Comment (DonFungible) [inline packages/core-sdk/test/integration/license.test.ts:40]: Should have separate tests -- one that waits and one that doesn't.

**Review by DonFungible — APPROVED** [review]
